CHANGELOG.rst

```diff
@@ -33,11 +34,13 @@ Removed
 - Drop Python 2.7, 3.4, 3.5 support.
 - Drop Pytest 4.x support.
 - Remove unittest2 and nose integrations. unittest2 and nose are not maintained anymore.
+- **BREAKING CHANGE**: Removed support for calling `once`, `twice`, `never`, and `mock` methods
+  without parentheses. This allows code completion and static analysis to work with these methods.
 
 Fixed
 #####
 
-- Fix ``with_args`` not working built-in functions and methods.
+- Fix `and_raise` allows invalid arguments for an exception.
```

Comment:
Why did you change this line ?

Reply:
This fix was backported to 0.10.x branch (https://github.com/flexmock/flexmock/blob/master/CHANGELOG.rst#release-0108) so this was a duplicate entry.